src/Raven.Studio/typescript/viewmodels/database/query/spatialQueryMap.ts

```diff
@@ -86,7 +86,7 @@ class spatialQueryMap extends viewModelBase {
             { color: spatialPolygonModel.colors[index % spatialPolygonModel.colors.length] }));
```

Comment:
for consistency, pls add fillOpacity for polygons here as well
=>
`{ color: spatialPolygonModel.colors[index % spatialPolygonModel.colors.length], fillOpacity: 0.4 }));`

Reply:
Added